bot.py

```diff
@@ -20,11 +20,17 @@
 async def dream(ctx, *, prompt):
     """Generate an image from a text prompt using the stable-diffusion model"""
     msg = await ctx.send(f"“{prompt}”\n> Generating...")
-
     model = replicate.models.get("stability-ai/stable-diffusion")
-    image = model.predict(prompt=prompt)[0]
-
+    try:
+        image = model.predict(prompt=prompt)[0]
+    except replicate.exceptions.ModelError as ex:
+        if "NSFW" in str(ex): # model output was classified as NSFW
+            await msg.edit(content="> NSFW detected. Please try again.")
```

Comment:
Maybe we should also include the original prompt in this message, so people know which one(s) failed, in case multiple simultaneous predictions are in flight.